pkg/plugins/plugins.go

```diff
@@ -403,6 +392,17 @@ func (s *PluginScanner) validateSignature(plugin *PluginBase) *PluginError {
 			"state", plugin.Signature)
 	}
 
+	// For the time being, we choose to only require back-end plugins to be signed
+	// NOTE: the state is calculated again when setting metadata on the object
+	if !plugin.Backend || !s.requireSigned {
+		return nil
+	}
+
+	if plugin.Signature == PluginSignatureValid {
+		s.log.Debug("Plugin has valid signature", "id", plugin.Id)
+		return nil
+	}
```

Comment:
I think this one should be moved back, since the check for this case is [here](https://github.com/grafana/grafana/blob/master/pkg/plugins/plugins.go#L396).